bundles/AdminBundle/DependencyInjection/Configuration.php

```diff
@@ -267,4 +254,26 @@ protected function buildDocumentsNode()
 
         return $documentsNode;
     }
+
+    protected function addNotificationsNode(): ArrayNodeDefinition|NodeDefinition
+    {
+        $treeBuilder = new TreeBuilder('notifications');
+        $notificationsNode = $treeBuilder->getRootNode();
+
+        $notificationsNode
+            ->addDefaultsIfNotSet()
+            ->children()
+                ->arrayNode('check_new_notification')
+                    ->canBeDisabled()
+                    ->children()
+                        ->integerNode('interval')
+                            ->defaultValue(30000)
```

Comment:
How about changing this to seconds? I don't think millisecond precision is needed here, and it's easier for the user in seconds.

Reply:
updated.